### DISCUSSION TITLE
RSS feed link

### DISCUSSION THREAD

**Discussion by mao1910** [dailydotdev/daily#976] [Feature Requests]
I might be mistaken, but I couldn't find a link for My Feed. Assuming it does not exist, could you create a way to make it available in the future? I'm sure some people would like to be able to get their hands on theirs to publish it (say in the Blog Activity for the GitHub Readme profiles for instance).

**Comment by idoshamun** [comment]
Hi @mao1910,

We don't offer RSS links for the feed, but you have one for your bookmarks.
https://docs.daily.dev/docs/integrations/sharing-bookmarks
